README.md

```diff
@@ -1,2 +1,55 @@
-# pharmacy-data-etl
-ETL to retrieve pharmacy data from NHS Organisations API
+# Pharmacy Data ETL
+ETL to retrieve Pharmacy information from [NHS Organisation API](http://api.nhs.uk/organisations) based on listings in [NHS Choices Syndication](http://www.nhs.uk/aboutNHSChoices/professionals/syndication/Pages/Webservices.aspx)
+and store as JSON.
+
+## Run process
+
+In order for the process to access the syndication feed an API key is required.
+Details of registration are available on
+[NHS Choices](http://www.nhs.uk/aboutNHSChoices/professionals/syndication/Pages/Webservices.aspx).
+The application needs the api key available within the environment as the variable `SYNDICATION_API_KEY`.
+
+The ETL retrieves the ODS codes for all Pharmacies from the Syndication API, then visits the organisation API to obtain full pharmacy information.
+
+Once the initial scan is complete, failed pharmacies will be revisited. ODS codes for records still failing after the second attempt are listed in a `summary.json` file.
+
+Running `scripts/start` will bring up a docker container hosting a web server and initiate the scrape at a scheduled time, GMT.
+The default is 11pm. To test locally set an environment variable `ETL_SCHEDULE` to a new time,
+i.e. `export ETL_SCHEDULE='25 15 * * *'` to start the processing a 3:25pm. Note: the container time is GMT and does not take account of daylight saving, you may need to subtract an hour from the time if it is currently BST.
+
+Further details on node-schedule available [here](https://www.npmjs.com/package/node-schedule)
+
+A successful scrape will result in the file `pharmacy-data.json` being written to the `output` folder. This file will also be uploaded to the Azure storage location specified in the environmental variables. The file will be uploaded twice, once to overwrite the current file at `pharmacy-data.json` and another date-stamped file at `YYYY-MM-DD-pharmacy-data.json`.
```

Comment:
I'd like to be able to inspect the generated JSON on Azure. Ideally I'd like to use the Azure Storage REST API to GET the files in a container? Whichever is best it would be worth documenting.